test/services/UpdateAddressResponseSpec.scala

```diff
@@ -46,27 +49,27 @@ class UpdateAddressResponseSpec extends BaseSpec with I18nSupport with MockitoSu
   def genericFunc(): Result =
     Ok
 
-  "UpdateAddressResponse.response" should {
+  "UpdateAddressResponse.response" must {
     "return the block result for UpdateAddressSuccessResponse" in {
-      val result = UpdateAddressSuccessResponse.response(genericFunc)
-      status(result) shouldBe OK
+      val result = Future(UpdateAddressSuccessResponse.response(genericErrors, genericFunc))
```

Comment:
Why has Future been wrapped around this?

Reply:
Done